src/app/NotFound/NotFound.tsx

```diff
@@ -0,0 +1,12 @@
+import * as React from 'react';
+import { NavLink } from 'react-router-dom';
+import { Alert, PageSection } from '@patternfly/react-core';
+
+export function NotFound() {
+  return (
+    <PageSection>
+      <Alert variant="danger" title="404! This view hasn't been created yet." /><br />
+      <NavLink to="/" className="pf-c-nav__link">Back to Dashboard</NavLink>
```

Comment:
Question: So it looks like navigating back to Dashboard is missing something, probably an aspect of state? The reason, when you navigate back to `dashboard` its looking like the `pf-m-current` class isn't being applied to `dashboard` or removed from `bad route` in the left nav

  
![Jul-03-2019 15-29-14](https://user-images.githubusercontent.com/3761375/60621866-d8dfeb80-9dac-11e9-97d1-09fed991d28b.gif)

Reply:
Should be fixed in the latest push